src/BlazorWebView/src/WindowsForms/BlazorWebView.cs

```diff
@@ -177,7 +178,7 @@ private void StartWebViewCoreIfPossible()
 			}
 			else
 			{
-				appRootDir = Environment.CurrentDirectory;
+			}
```

Comment:
This doesn't look like it will build, there is an extra `}`.

Should it be:
```suggestion
				appRootDir = AppContext.BaseDirectory;
```

Reply:
# Will make correction